knimin/lib/util.py

```diff
@@ -198,6 +198,26 @@ def categorize_age(x):  # noqa
     return age_cat
 
 
+def correct_age(age, height, weight, etoh):
+    """Infers incorrect ages and incorrectly classified babies"""
+    # Checks the logic for age
+    if age >= 4 and age < 123:
```

Comment:
Alright! Makes sense.

Could you rename the function to something like `fix_babies_age` or something?